src/internal/Popover.js

```diff
@@ -140,11 +176,15 @@ export default class Popover extends Component {
     styleManager: PropTypes.object.isRequired,
   };
 
+  static getScale(value) {
```

Comment:
@nathanmarks That's the only `static` method I haven't removed.
The other best approach I can think of is making the test assertion less high level.